src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const PAGINATION = 'page';
```

Comment:
How about naming them `PAGE_KEY` & `CONTINUATION_KEY`? Not sure what I'd expect to get if I `import {PAGINATION} from 'eventbrite'`, but the string `'page'` seems surprising.